test/unittests/UtilityTest.php

```diff
@@ -577,7 +574,7 @@ public function testGetExistingVisitLabelsWithProjectID()
             ->method('pselect')
             ->with(
                 $this->stringContains(
-                    "AND (c.ProjectID IS NULL OR c.ProjectID=:ProjectID)"
+                    "AND (s.ProjectID IS NULL OR s.ProjectID=:ProjectID)"
```

Comment:
changes were made to resolve Failures

https://travis-ci.org/aces/Loris/jobs/572998062